src/api/gameApi.ts

```diff
@@ -6,14 +6,14 @@ import GameInstance from "../interfaces/GameInstance";
 
 export const getGame = async (id: string): Promise<Game> => {
     const response = await fetch(homeUrl + gamesBaseUrl + `${id}`, {
-        method: "GET",
```

Comment:
Did a bunch of drivebys to remove trailing commas while debugging JSON parsing issue, seemed like a good thing to leave in even though it wasn't the cause